lib/http.js

```diff
@@ -21,6 +21,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     profiles: basePath,
     profileAgents: `${profileAgentsPath}`,
     profileAgent: `${profileAgentPath}`,
+    profileAgentClaim: `${profileAgentsPath}/:profileAgentId/claim`,
```

Comment:
Can simplify to this:

```suggestion
    profileAgentClaim: `${profileAgentPath}/claim`,
```